services/quests/src/components/Dialogs.tsx

```diff
@@ -314,6 +314,13 @@ export class PublishingDialog extends React.Component<PublishingDialogProps, {}>
               onChange={(checked: boolean) => { handleMetadataChange(quest, 'expansionfuture', checked); }}>
             </Checkbox>
           </FormControl>
+          <FormControl className="fullWidth">
+            <Checkbox
+              label="Requires &quot;Of Wyrms &amp; Giants&quot; Booster"
+              value={metadata.get('expansionwyrmsgiants')}
+              onChange={(checked: boolean) => { handleMetadataChange(quest, 'expansionwyrmsgiants', checked); }}>
```

Comment:
I'm a little sad we have a string constant accessor only in this one place. I don't suppose as part of this PR you could refactor `handleMetadataChange` so it takes a Partial<Quest> and then use it to set one or multiple values?

Reply:
Yeah :/ I don't like doing big refactors in the same PR as adding functionality, though, since that makes it harder to tell what's a functionality change vs what's a no-op refactor